terraform/modules/paas/variables.tf

```diff
@@ -72,13 +75,15 @@ locals {
     staging   = "staging"
     sandbox   = "sandbox"
     research  = "research"
+    review    = "review"
```

Comment:
We're creating extra routes:
```apply-pr-6147                                         started           web:1/1     review.apply-for-teacher-training.education.gov.uk, apply-pr-6147.london.cloudapps.digital, review-assets.apply-for-teacher-training.service.gov.uk, apply-pr-6147.apps.internal, review.apply-for-teacher-training.service.gov.uk
```
I think it's related to this change. We could refactor and make these conditional.